backend/python/app/services/implementations/story_service.py

```diff
@@ -51,3 +52,37 @@ def create_story(self, story, content):
         db.session.refresh(new_story)
 
         return new_story
+
+    def get_story_translations(self, user_id, translate):
+        if translate:
+            return (
```

Comment:
can you wrap this in a try-except and log exceptions? (see `get_user_by_id `or a similar service fn)